content/shmem_reductions.tex

```diff
@@ -19,7 +19,7 @@
 \end{Fsynopsis}
 
 \paragraph{MAX}
-Performs a maximum function reduction across a set of processing elements (\acp{PE}).\newline
+Performs a maximum-function reduction across a set of processing elements (\acp{PE}).\newline
```

Comment:
@BryantLam How about this ? 

> "Computes a maximum value across a set of PEs"

Reply:
I agree with "maximum value" but was not sure if it was considered out of scope of a document edit. The intent is definitely of "maximum value" so I'm willing to change it.